tests/test_projectbuilder.py

```diff
@@ -165,30 +164,27 @@ def test_build_missing_backend(packages_path, distribution, tmpdir):
 
 
 def test_check_dependencies(mocker, test_flit_path):
-    mocker.patch('importlib.import_module', autospec=True)
     mocker.patch('pep517.wrappers.Pep517HookCaller.get_requires_for_build_sdist')
     mocker.patch('pep517.wrappers.Pep517HookCaller.get_requires_for_build_wheel')
-    mocker.patch('build.check_version', autospec=True)
 
     builder = build.ProjectBuilder(test_flit_path)
 
     side_effects = [
         [],
-        ['something'],
+        ['foo'],
```

Comment:
Why? This is unrelated and just dirties git blame.

Reply:
Because `importlib_metadata.Distribution` is mocked further up and `something` is would return a `Distribution` instance, i.e. the dependency would be satisfied.

Reply:
I've removed the assignment to `importlib_metadata` from the global scope and reverted this change.